pkg/config/types/v1alpha1/network_config.go

```diff
@@ -20,6 +20,7 @@ import (
 type NetworkConfig struct {
 	NetworkHostname   string           `yaml:"hostname,omitempty"`
 	NetworkInterfaces []machine.Device `yaml:"interfaces,omitempty"`
+	NameServers       []string         `yaml:"resolvers,omitempty"`
```

Comment:
Is `nameservers` a better public name here? I feel like it hints at what it is within `/etc/resolv.conf` better.

Reply:
Just to be clear I meant the yaml tag: `yaml:”nameservers”`